src/app/components/Transcript/index.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { render } from '../react-testing-library-with-providers';
+import transcriptFixture from './fixture.json';
+import Transcript from './index';
+
+describe('Transcript Component', () => {
+  it('should match snapshot (temp)', () => {
```

Comment:
Shall we just remove this test altogether, and rely on either a Chromatic snapshot, or explicitly test the elements we expect to see?